portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -104,6 +116,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     response.sendRedirect("/index.html");
   }
 
+  /** Run sentiment analysis and return sentiment score */
+  private double getSentimentScore(String message) throws IOException {
+    Document doc =
+        Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
```

Comment:
Could this be a field variable, so it's only created once? Creating the service client is likely an expensive operation.

Reply:
When I tried to make it a field variable I got this error: error: unreported exception java.io.IOException; must be caught or declared to be thrown